schema.graphql

```diff
@@ -2766,6 +2769,7 @@ input UserRegisterInput {
   password: String!
   description: String
   codeId: ID!
+  referralCode: String
```

Comment:
Note that `userRegister` and `userLogin`  are replaced by `emailLogin` .  And users who sign up using wallet  (`walletLogin`)or social accounts  (`socialLogin`) can also have referral code ?